src/plugins/blocksmith/components/google-map-block-preview/index.jsx

```diff
@@ -11,7 +11,7 @@ const GoogleMapBlockPreview = ({ block, width }) => {
       title="google-map-block-preview"
       width={width}
       height={width / (16 / 9)}
-      src={`https://www.google.com/maps/embed/v1/place?key=${GOOGLE_MAPS_API_KEY}${zoomParam}&maptype=${block.get('maptype') || 'roadmap'}&q=${block.get('q') || ''}${block.has('center') ? `&center=${block.get('center').getLatitude()},${block.get('center').getLongitude()}` : ''}`}
+      src={`https://www.google.com/maps/embed/v1/place?key=${GOOGLE_MAPS_API_KEY}${zoomParam}&maptype=${block.get('maptype') || 'roadmap'}&q=${block.get('q') || ''}`}
```

Comment:
if both is there center takes precedence.  I think it should be removed cause on web we don't even use it

Reply:
@bhsiao55 - if the map is going to be more precise when the *center* field is part of the URL, then let's leave it in there. Otherwise, please remove it.